doc/classes/CollisionObject2D.xml

```diff
@@ -195,6 +197,7 @@
 			<argument index="1" name="enable" type="bool">
 			</argument>
 			<description>
+				Sets one way collision of the given shape owner.
```

Comment:
I think it would help to have a brief explanation of what "one way collision" means. See CollisionPolygon2D.